tests/opentelemetry-remote-collector-tests.bats

```diff
@@ -1,87 +0,0 @@
-#!/usr/bin/env bats
-
-# UI access:
-# kubectl port-forward -n prometheus --address 0.0.0.0 svc/prometheus-operated 9090
-# kubectl port-forward -n jaeger svc/my-open-telemetry-query 16686:16686
```

Comment:
Could we keep these comments in opentelemetry-tests.bats? I always find these things useful.

Reply:
jj, [they are there](https://github.com/kubewarden/kubewarden-end-to-end-tests/pull/132/files#diff-f5c45b9f2b57a1e989aa6d37719f7c4d3321f2378229f2753bf74f3aea8ae895L3), otel-remote was just copy & paste of this file.

Reply:
true! resolving